src/wallet/keystore.ts

```diff
@@ -0,0 +1,58 @@
+import {makeSurePathExists, resolveHome} from '../core/utils';
+import path from 'path';
+import config from 'config';
+import Wallet, {hdkey} from 'ethereumjs-wallet';
+import bip39 from 'bip39';
+
+const keystorePath = path.join(resolveHome(config.get('datadir')), config.get('wallet.keystore_path'));
+
+function getWalletFactory() {
+    let wallet: Wallet | undefined;
+    return (): Wallet => {
+        if (!wallet) {
+            const keystorePath = path.join(resolveHome(config.get('datadir')), config.get('wallet.keystore_path'));
+            makeSurePathExists(keystorePath);
+            const secretPhrase = require(path.join(keystorePath, 'key.json')).phrase;
+            const hdwallet = hdkey.fromMasterSeed(bip39.mnemonicToSeedSync(secretPhrase));
+            wallet = hdwallet.getWallet();
+        }
+        return wallet;
+    };
+}
+
+function getArweaveKeyFactory() {
+    let arweaveKey: string|undefined|null = null;
+    return (): string|undefined => {
+        if (arweaveKey === null) {
+            try {
+                arweaveKey = require(path.join(keystorePath, 'arweave.json'));
+            }catch(e){
```

Comment:
Could you please check the linter rules for the syntax like that? I was under impression that it should require spaces between the brackets here, so that the statement should look like
```
} catch (e) {
```